data/src/test/java/org/apache/iceberg/data/avro/TestSingleMessageEncoding.java

```diff
@@ -87,8 +86,10 @@ public void testByteBufferRoundTrip() throws Exception {
 
     Record copy = decoder.decode(encoder.encode(V2_RECORDS.get(0)));
 
-    Assert.assertTrue("Copy should not be the same object", copy != V2_RECORDS.get(0));
-    Assert.assertEquals("Record should be identical after round-trip", V2_RECORDS.get(0), copy);
+    Assertions.assertThat(copy).as("Copy should not be the same object").isNotSameAs(V2_RECORDS.get(0));
+    Assertions.assertThat(V2_RECORDS.get(0))
+        .as("Record should be identical after round-trip")
+        .isEqualTo(copy);
```

Comment:
`V2_RECORDS.get(0)` is the expected and `copy` is the actual, so the assertion needs to be the other way around here and in a bunch of other places